roles/node/templates/testnet/rollup.toml.j2

```diff
@@ -0,0 +1,54 @@
+[da]
+# The JWT used to authenticate with the celestia light client. Instructions for generating this token can be found in the README
+celestia_rpc_auth_token = "{{ vault.testnet.auth_token }}"
+# The address of the *trusted* Celestia light client to interact with
+celestia_rpc_address = "{{ vault.testnet.rpc_address }}"
+# The largest response the rollup will accept from the Celestia node. Defaults to 100 MB
+max_celestia_response_body_size = 104_857_600
+# The maximum time to wait for a response to an RPC query against Celestia node. Defaults to 60 seconds.
+celestia_rpc_timeout_seconds = 60
+signer_address = "celestia1xetdm9tl7c08vatjnujnql2xtm8h34v7hhvgm3"
+
+
+[storage]
+# The path to the rollup's data directory. Paths that do not begin with `/` are interpreted as relative paths.
+path = "{{ node_data_dir }}"
+
+# We define the rollup's genesis to occur at block number `start_height`. The rollup will ignore
+# any blocks before this height
+[runner]
+genesis_height = 6808386
+da_polling_interval_ms = 1000
+
+[runner.http_config]
+bind_host = "127.0.0.1"
+bind_port = {{ spicenet[type].ports.http_port }}
+
+[monitoring]
+telegraf_address = "127.0.0.1:8094"
```

Comment:
we wont have a telegraf running ? or will the binary run it internally ? if yes then we might wanna update the port